realm/src/main/java/io/realm/BaseRealm.java

```diff
@@ -494,8 +509,39 @@ static synchronized void releaseFileReference(RealmConfiguration configuration)
         globalRealmFileReferenceCounter.put(canonicalPath, refCount - 1);
     }
 
+    // Public because of migrations
+    @Deprecated
+    public Table getTable(String className) {
+        className = Table.TABLE_PREFIX + className;
+        Table table = dynamicClassToTable.get(className);
+        if (table == null) {
+            if (!sharedGroupManager.hasTable(className)) {
+                throw new IllegalArgumentException("The type " + className + " doesn't exist in this Realm.");
```

Comment:
I changed it to `class` for now. It is in line with the vocabulary used elsewhere currently